docs/source/use_cases/load_data/multi_sample_variants.rst

```diff
@@ -0,0 +1,75 @@
+
+Overview
+--------
+
+Suppose you have imported your single-sample files to the Variants API and exported them to BigQuery.  Let's refer to this original table as the "genome calls" table. It contains *all* reference calls and variant calls.
+
+To facilitate variant-centric analysis like we see in the `BigQuery 1,000 Genomes samples <https://github.com/googlegenomics/bigquery-examples/blob/master/1000genomes/README.md>`_, we can generate a second table, the "multi-sample variants" table.  The multi-sample variants table resembles a multi-sample VCF file. In this table:
+
+* every variant record includes calls for *all* callsets
+* variants which contained *only reference calls for all callsets* are omitted
+
+Motivation
+----------
+
+Data from source files in `genome VCF`_ (gVCF) format or in Complete Genomics format can be challenging to query due to the presence of non-variant segment records.
+
+For example to lookup `rs9536314 <http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=rs9536314>`_ in the Klotho gene, the `WHERE` clause
+
+.. code::
+
+    WHERE
+      reference_name = 'chr13'
+      AND start = 33628137
+
+becomes
+
+.. code::
+
+    WHERE
+      reference_name = 'chr13'
+      AND start <= 33628137
+      AND end >= 33628138
+
+to capture not only that variant, but any other records that overlap that genomic position.
+
+Suppose we want to calculate an aggregate for a particular variant, such as the number of samples with the variant on one or both alleles and of samples that match the reference?  The WHERE clause above will do the trick.  But then suppose we want to do this for all SNPs in our dataset?
+
+Examples
+--------
+
+There are a few ways to generate the multi-sample variants table for use in variant-centric analyses such as :doc:`/use_cases/analyze_variants/gwas`:
+
+* Use a cluster computing job to transform data with non-variant segments to variant-only data with calls from non-variant-segments merged into the variants with which they overlap. This is currently done only for SNP variants. Indels and structural variants are left as-is.
+
+  * see the `Dataflow example <https://github.com/googlegenomics/codelabs/tree/master/Java/PlatinumGenomes-variant-transformation>`_
+  * see the `Hadoop Streaming example <https://github.com/googlegenomics/codelabs/tree/master/Python/PlatinumGenomes-variant-transformation>`_
+
+* Use BigQuery to transform the data, materialize the result to a new table
+
+  * https://github.com/googlegenomics/bigquery-examples/tree/master/pgp/data-stories/schema-comparisons#motivation
+
+A note about scaling: as the number of samples increases, so does the number of private and rare variants.  At a certain point there are many, many rows with mostly 0/0 genotypes.  We are experimenting with alternate transformations.  Comment on `this issue <https://github.com/googlegenomics/codelabs/issues/52>`_ if you want a pointer to the most recent prototype.
```

Comment:
Nit: this paragraph double-spaces after periods. Other paragraphs use the more modern single-space after paragraph.